#!/usr/bin/env python3

PACKAGE = "waypoint_maker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("xMinRubberCone", double_t, 0, "xMinRubberCone", 0.05, 0, 10);
gen.add("xMaxRubberCone", double_t, 0, "xMaxRubberCone", 0.55, 0, 10);
gen.add("yMinRubberCone", double_t, 0, "yMinRubberCone", 0.05, 0, 10);
gen.add("yMaxRubberCone", double_t, 0, "yMaxRubberCone", 0.55, 0, 10);
gen.add("zMinRubberCone", double_t, 0, "zMinRubberCone", 0.05, 0, 10);
gen.add("zMaxRubberCone", double_t, 0, "zMaxRubberCone", 0.85, 0, 10);

exit(gen.generate(PACKAGE, "waypoint_maker_static_obstacle", "waypointMaker"))
